podvm-payload/Dockerfile

```diff
@@ -39,6 +39,12 @@ RUN if command -v subscription-manager; then \
     fi
 RUN dnf clean packages && dnf install -y git rust cargo perl-File-Compare perl-FindBin cmake gcc-c++ perl protobuf-compiler clang-devel device-mapper-devel tpm2-tss-devel
 
+# Install tdx deps from custom copr for now
+RUN dnf install -y 'dnf-command(copr)' && \
+    dnf copr enable -y berrange/sgx-ng rhel-9-x86_64 && \
```

Comment:
I'm not sure how that will work with hermetic builds, but I'm sure I can find out.
Is that only x86_64? Should we ignore that for s390x?

Reply:
Yeah, this is only for x86_64 and doesn't apply for s390x.

Reply:
I'm still thinking on CPaaS mode, so this would break s390x builds as @littlejawa said.

Not sure how reusable Konflux would be on that Dockerfile for both platforms. But if @littlejawa plans to reuse the same file, maybe we should just add a conditional here, to avoid this block on s390x?

Reply:
I'll add a conditional